sormas-app/app/src/main/java/de/symeda/sormas/app/backend/user/UserDao.java

```diff
@@ -73,6 +75,34 @@ public User getByUsername(String username) {
 		}
 	}
 
+	public List<User> getAllInJurisdiction() {
+		try {
+			QueryBuilder builder = queryBuilder();
```

Comment:
Again, I think we'll run into problems here. We can't restrict the list of users that are retrieved because this will lead to empty mandatory fields or empty values where a value was supposed to be. I think we need a solution that actually pseudonymises the data here instead of filtering the list of possible options.